client/Tutorial/TutorialSteps.ts

```diff
@@ -21,41 +22,108 @@ function getLocation(element: HTMLElement) {
+    RaiseSelector: ".c-tab:nth-child(2) ",
     AwaitAction: "SelectCharactersTab",
     CalculatePosition: elements => {
       const element = _.last(elements);
       const location = getLocation(element);
       const left = location.left + location.width + 10;
       const top = location.top + 5;
       return { left, top };
-    }
+    },
+    HighlightSelector: ".c-tab:nth-child(2)",
   },
+  // step 3 - add sample characters
   {
     Message:
       "It's easy to add your own player characters to Improved Initiative. For now, <strong>add a few sample characters</strong>.",
     RaiseSelector: ".left-column, .combatants",
+    AwaitAction: "PlayerCharacterAdded",
     CalculatePosition: elements => {
       const location = getLocation(elements[0]);
       const left = location.left + location.width + 10;
       const top = location.top + 200;
       return { left, top };
-    }
+    },
+    HighlightSelector: ".libraries",
+  },
+  // step 4 - select encounters tab
+  {
+    Message:
+      "Select the <strong>Encounters</strong> tab at the top of the library.",
+    RaiseSelector: ".c-tab:nth-child(3)",
+    AwaitAction: "SelectEncountersTab",
+    CalculatePosition: elements => {
+      const element = _.last(elements);
+      const location = getLocation(element);
+      const left = location.left + location.width + 10;
+      const top = location.top + 5;
+      return { left, top };
+    },
+    HighlightSelector: ".c-tab:nth-child(3)",
+  },
+  // step 5 - encounter tab explanation
+  {
+    Message:
+      "The Encounters tab is where you can <strong>save and load</strong> encounters.",
+    RaiseSelector: ".left-column",
+    CalculatePosition: elements => {
+      const element = _.last(elements);
+      const location = getLocation(element);
+      const left = location.left + location.width + 10;
+      const top = location.top + 5;
+      return { left, top };
+    },
+    HighlightSelector: ".libraries",
+  },
+  // step 6 - select spells tab
+  {
+    Message:
+      "Select the <strong>Spells</strong> tab at the top of the library.",
+    RaiseSelector: ".c-tab:nth-child(4)",
+    AwaitAction: "SelectSpellsTab",
+    CalculatePosition: elements => {
+      const element = _.last(elements);
+      const location = getLocation(element);
+      const left = location.left + location.width + 10;
+      const top = location.top + 5;
+      return { left, top };
+    },
+    HighlightSelector: ".c-tab:nth-child(4)",
+  },
+  // step 7 - spell tab explanation
+  {
+    Message:
+      "The Spells tab is where you can <strong>select and view</strong> spell information.",
+    RaiseSelector: ".left-column",
+    CalculatePosition: elements => {
+      const element = _.last(elements);
+      const location = getLocation(element);
+      const left = location.left + location.width + 10;
+      const top = location.top + 5;
+      return { left, top };
+    },
+    HighlightSelector: ".libraries",
   },
```

Comment:
I don't think these steps need to be included in the basic tutorial. They may be distracting from the core functionality that we want to teach the users.